testgrid/config.md

```diff
@@ -57,6 +57,67 @@ Open or create a Testgrid config file [(example)][configuration] in your favorit
 2. Add those testgroups to one or more tabs in one or more dashboards
 3. Consider using dashboard groups if multiple dashboards are needed.
 
+### Defaults
+
+#### Overall Default.yaml
+
+For testgrid.k8s.io there is a default.yaml file that contains configuration that will apply to all other testgroups and dashboard_tabs. 
+This will rarely need to be changed, but to override these defaults, or to have defaults for your own testgroups/dashboard tabs you can use
+a directory default. 
+
+#### Directory Default.yaml
+
+If you want to override the default.yaml for configs in a directory, you can add a file named "default.yaml" and instead of applying the overall default to 
+those config files, it will apply this default file instead.
+
+What this default will NOT apply to:
+1. Configs in subdirectories
```

Comment:
nit: unordered lists should use bullets, not numbers

Reply:
Done